### DISCUSSION TITLE
Support for Qwik.js

### DISCUSSION THREAD

**Discussion by md-abdul-halim-rafi** [mantinedev/mantine#4226] [Feature requests]
The Mantine UI library is absolutely fantastic. But, there are no Qwik integration guides in the documentation. I would love to request the Qwik integration support 🙏

**Comment by rtivital** [comment]
Qwik is not React, Mantine only works with React-based frameworks, it is not planned to support other frameworks

**Reply by md-abdul-halim-rafi** [reply]
Understood. Then which component libraries we can use in Qwik?
